core/shared/src/main/scala-3/me/shadaj/scalapy/py/None.scala

```diff
@@ -0,0 +1,9 @@
+package me.shadaj.scalapy
+
+import me.shadaj.scalapy.interpreter.CPythonInterpreter
+import me.shadaj.scalapy.py.Any
+
+trait PyNone {
+  @py.native class None extends Any
+  val None = Any.populateWith(CPythonInterpreter.noneValue).as[None]
```

Comment:
Static `None` value can be shared between Scala 2/3

Reply:
<details>
  <summary>Got compile errors:</summary>

```
[error] -- [E007] Type Mismatch Error: ../scalapy/core/shared/src/main/scala/me/shadaj/scalapy/py/package.scala:18:13 
[error] 18 |  val None = Any.populateWith(CPythonInterpreter.noneValue).as[None]
[error]    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error]    |             Found:    me.shadaj.scalapy.py.None
[error]    |             Required: object me.shadaj.scalapy.py.package.None
[error] -- [E007] Type Mismatch Error: ../scalapy/core/shared/src/main/scala/me/shadaj/scalapy/py/package.scala:27:31 
[error] 27 |      ref.as[Dynamic].__exit__(None, None, None)
[error]    |                               ^^^^
[error]    |Found:    (me.shadaj.scalapy.py.None : object me.shadaj.scalapy.py.package.None²)
[error]    |Required: me.shadaj.scalapy.py.Any
[error]    |
[error]    |where:    None  is a value in package object py
[error]    |          None² is a object in trait PyNone
[error] -- [E007] Type Mismatch Error: ../scalapy/core/shared/src/main/scala/me/shadaj/scalapy/py/package.scala:27:37 
[error] 27 |      ref.as[Dynamic].__exit__(None, None, None)
[error]    |                                     ^^^^
[error]    |Found:    (me.shadaj.scalapy.py.None : object me.shadaj.scalapy.py.package.None²)
[error]    |Required: me.shadaj.scalapy.py.Any
[error]    |
[error]    |where:    None  is a value in package object py
[error]    |          None² is a object in trait PyNone
[error] -- [E007] Type Mismatch Error: ../scalapy/core/shared/src/main/scala/me/shadaj/scalapy/py/package.scala:27:43 
[error] 27 |      ref.as[Dynamic].__exit__(None, None, None)
[error]    |                                           ^^^^
[error]    |Found:    (me.shadaj.scalapy.py.None : object me.shadaj.scalapy.py.package.None²)
[error]    |Required: me.shadaj.scalapy.py.Any
[error]    |
[error]    |where:    None  is a value in package object py
[error]    |          None² is a object in trait PyNone
```
</details>
Very confused about this case.

Reply:
We can share the code in this way: 
```
object NoneFacade {
  @native class None extends Any
}
```
and then:
```
trait PyNone {
  val None = Any.populateWith(CPythonInterpreter.noneValue).as[NoneFacade.None]
}
```
and in the package py:
```
package object py extends PyMacros with PyNone {
  def module(name: String) = Module(name)
  def module(name: String, subname: String) = Module(name, subname)

  type NoneOr[T] = NoneFacade.None | T

```

Reply:
It seems that @jlareck 's idea works and furthermore we can simply put `val None = Any.populateWith(CPythonInterpreter.noneValue).as[NoneFacade.None]`into `package object py` without defining a trait.